diff --git a/node_modules/remix/esm/index.js b/node_modules/remix/esm/index.js
index aacd9d1..b173819 100644
--- a/node_modules/remix/esm/index.js
+++ b/node_modules/remix/esm/index.js
@@ -1,5 +1,5 @@
 /**
- * remix v1.2.3
+ * @remix-run/server-runtime v1.2.3
  *
  * Copyright (c) Remix Software Inc.
  *
@@ -8,4 +8,17 @@
  *
  * @license MIT
  */
-throw new Error("Did you forget to run `remix setup` for your platform?");
+export { createCookie, createCookieSessionStorage, createMemorySessionStorage, createSession, createSessionStorage, isCookie, isSession, json, redirect } from '@remix-run/server-runtime';
+
+/**
+ * @remix-run/react v1.2.3
+ *
+ * Copyright (c) Remix Software Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE.md file in the root directory of this source tree.
+ *
+ * @license MIT
+ */
+export { Form, Link, Links, LiveReload, Meta, NavLink, Outlet, PrefetchPageLinks, RemixBrowser, RemixServer, Scripts, ScrollRestoration, useActionData, useBeforeUnload, useCatch, useFetcher, useFetchers, useFormAction, useHref, useLoaderData, useLocation, useMatches, useNavigate, useNavigationType, useOutlet, useOutletContext, useParams, useResolvedPath, useSearchParams, useSubmit, useTransition } from '@remix-run/react';
+
diff --git a/node_modules/remix/index.d.ts b/node_modules/remix/index.d.ts
index e69de29..3783d81 100644
--- a/node_modules/remix/index.d.ts
+++ b/node_modules/remix/index.d.ts
@@ -0,0 +1,9 @@
+export { createCloudflareKVSessionStorage } from "@remix-run/cloudflare-workers";
+
+export type { ServerBuild, ServerEntryModule, HandleDataRequestFunction, HandleDocumentRequestFunction, CookieParseOptions, CookieSerializeOptions, CookieSignatureOptions, CookieOptions, Cookie, AppLoadContext, AppData, EntryContext, LinkDescriptor, HtmlLinkDescriptor, PageLinkDescriptor, ErrorBoundaryComponent, ActionFunction, HeadersFunction, LinksFunction, LoaderFunction, MetaDescriptor, HtmlMetaDescriptor, MetaFunction, RouteComponent, RouteHandle, RequestHandler, SessionData, Session, SessionStorage, SessionIdStorageStrategy, } from "@remix-run/server-runtime";
+export { createCookie, isCookie, createSession, isSession, createSessionStorage, createCookieSessionStorage, createMemorySessionStorage, json, redirect, } from "@remix-run/server-runtime";
+
+export type { RemixBrowserProps, FormProps, SubmitOptions, SubmitFunction, FormMethod, FormEncType, RemixServerProps, ShouldReloadFunction, ThrownResponse, LinkProps, NavLinkProps, } from "@remix-run/react";
+export { RemixBrowser, Meta, Links, Scripts, Link, NavLink, Form, PrefetchPageLinks, ScrollRestoration, LiveReload, useFormAction, useSubmit, useTransition, useFetcher, useFetchers, useCatch, useLoaderData, useActionData, useBeforeUnload, useMatches, RemixServer, } from "@remix-run/react";
+export { Outlet, useHref, useLocation, useNavigate, useNavigationType, useOutlet, useParams, useResolvedPath, useSearchParams, useOutletContext, } from "@remix-run/react";
+
diff --git a/node_modules/remix/index.js b/node_modules/remix/index.js
index 0b4073f..30074c3 100644
--- a/node_modules/remix/index.js
+++ b/node_modules/remix/index.js
@@ -1,5 +1,5 @@
 /**
- * remix v1.2.3
+ * @remix-run/server-runtime v1.2.3
  *
  * Copyright (c) Remix Software Inc.
  *
@@ -10,4 +10,189 @@
  */
 'use strict';
 
-throw new Error("Did you forget to run `remix setup` for your platform?");
+Object.defineProperty(exports, '__esModule', { value: true });
+
+var serverRuntime = require('@remix-run/server-runtime');
+
+
+
+Object.defineProperty(exports, 'createCookie', {
+  enumerable: true,
+  get: function () { return serverRuntime.createCookie; }
+});
+Object.defineProperty(exports, 'createCookieSessionStorage', {
+  enumerable: true,
+  get: function () { return serverRuntime.createCookieSessionStorage; }
+});
+Object.defineProperty(exports, 'createMemorySessionStorage', {
+  enumerable: true,
+  get: function () { return serverRuntime.createMemorySessionStorage; }
+});
+Object.defineProperty(exports, 'createSession', {
+  enumerable: true,
+  get: function () { return serverRuntime.createSession; }
+});
+Object.defineProperty(exports, 'createSessionStorage', {
+  enumerable: true,
+  get: function () { return serverRuntime.createSessionStorage; }
+});
+Object.defineProperty(exports, 'isCookie', {
+  enumerable: true,
+  get: function () { return serverRuntime.isCookie; }
+});
+Object.defineProperty(exports, 'isSession', {
+  enumerable: true,
+  get: function () { return serverRuntime.isSession; }
+});
+Object.defineProperty(exports, 'json', {
+  enumerable: true,
+  get: function () { return serverRuntime.json; }
+});
+Object.defineProperty(exports, 'redirect', {
+  enumerable: true,
+  get: function () { return serverRuntime.redirect; }
+});
+
+/**
+ * @remix-run/react v1.2.3
+ *
+ * Copyright (c) Remix Software Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE.md file in the root directory of this source tree.
+ *
+ * @license MIT
+ */
+'use strict';
+
+Object.defineProperty(exports, '__esModule', { value: true });
+
+var react = require('@remix-run/react');
+
+
+
+Object.defineProperty(exports, 'Form', {
+  enumerable: true,
+  get: function () { return react.Form; }
+});
+Object.defineProperty(exports, 'Link', {
+  enumerable: true,
+  get: function () { return react.Link; }
+});
+Object.defineProperty(exports, 'Links', {
+  enumerable: true,
+  get: function () { return react.Links; }
+});
+Object.defineProperty(exports, 'LiveReload', {
+  enumerable: true,
+  get: function () { return react.LiveReload; }
+});
+Object.defineProperty(exports, 'Meta', {
+  enumerable: true,
+  get: function () { return react.Meta; }
+});
+Object.defineProperty(exports, 'NavLink', {
+  enumerable: true,
+  get: function () { return react.NavLink; }
+});
+Object.defineProperty(exports, 'Outlet', {
+  enumerable: true,
+  get: function () { return react.Outlet; }
+});
+Object.defineProperty(exports, 'PrefetchPageLinks', {
+  enumerable: true,
+  get: function () { return react.PrefetchPageLinks; }
+});
+Object.defineProperty(exports, 'RemixBrowser', {
+  enumerable: true,
+  get: function () { return react.RemixBrowser; }
+});
+Object.defineProperty(exports, 'RemixServer', {
+  enumerable: true,
+  get: function () { return react.RemixServer; }
+});
+Object.defineProperty(exports, 'Scripts', {
+  enumerable: true,
+  get: function () { return react.Scripts; }
+});
+Object.defineProperty(exports, 'ScrollRestoration', {
+  enumerable: true,
+  get: function () { return react.ScrollRestoration; }
+});
+Object.defineProperty(exports, 'useActionData', {
+  enumerable: true,
+  get: function () { return react.useActionData; }
+});
+Object.defineProperty(exports, 'useBeforeUnload', {
+  enumerable: true,
+  get: function () { return react.useBeforeUnload; }
+});
+Object.defineProperty(exports, 'useCatch', {
+  enumerable: true,
+  get: function () { return react.useCatch; }
+});
+Object.defineProperty(exports, 'useFetcher', {
+  enumerable: true,
+  get: function () { return react.useFetcher; }
+});
+Object.defineProperty(exports, 'useFetchers', {
+  enumerable: true,
+  get: function () { return react.useFetchers; }
+});
+Object.defineProperty(exports, 'useFormAction', {
+  enumerable: true,
+  get: function () { return react.useFormAction; }
+});
+Object.defineProperty(exports, 'useHref', {
+  enumerable: true,
+  get: function () { return react.useHref; }
+});
+Object.defineProperty(exports, 'useLoaderData', {
+  enumerable: true,
+  get: function () { return react.useLoaderData; }
+});
+Object.defineProperty(exports, 'useLocation', {
+  enumerable: true,
+  get: function () { return react.useLocation; }
+});
+Object.defineProperty(exports, 'useMatches', {
+  enumerable: true,
+  get: function () { return react.useMatches; }
+});
+Object.defineProperty(exports, 'useNavigate', {
+  enumerable: true,
+  get: function () { return react.useNavigate; }
+});
+Object.defineProperty(exports, 'useNavigationType', {
+  enumerable: true,
+  get: function () { return react.useNavigationType; }
+});
+Object.defineProperty(exports, 'useOutlet', {
+  enumerable: true,
+  get: function () { return react.useOutlet; }
+});
+Object.defineProperty(exports, 'useOutletContext', {
+  enumerable: true,
+  get: function () { return react.useOutletContext; }
+});
+Object.defineProperty(exports, 'useParams', {
+  enumerable: true,
+  get: function () { return react.useParams; }
+});
+Object.defineProperty(exports, 'useResolvedPath', {
+  enumerable: true,
+  get: function () { return react.useResolvedPath; }
+});
+Object.defineProperty(exports, 'useSearchParams', {
+  enumerable: true,
+  get: function () { return react.useSearchParams; }
+});
+Object.defineProperty(exports, 'useSubmit', {
+  enumerable: true,
+  get: function () { return react.useSubmit; }
+});
+Object.defineProperty(exports, 'useTransition', {
+  enumerable: true,
+  get: function () { return react.useTransition; }
+});
+
